storage/persistent_storage/persistent-storage-cinder.adoc

```diff
@@ -34,3 +34,12 @@ include::modules/persistent-storage-cinder-creating-pv.adoc[leveloffset=+2]
 include::modules/persistent-storage-cinder-pv-format.adoc[leveloffset=+2]
 
 include::modules/persistent-storage-cinder-volume-security.adoc[leveloffset=+2]
+
+include::modules/persistent-storage-cinder-maximum-volume.adoc[leveloffset=+2]
```

Comment:
I'm not sure it makes sense to just append this to the end after the procedure. Wouldn't someone want to know this before completing the procedure?

Reply:
This is consistent to what we have for EBS https://docs.openshift.com/container-platform/4.6/storage/persistent_storage/persistent-storage-aws.html#maximum-number-of-ebs-volumes-on-a-node_persistent-storage-aws. 
But I do agree that it could be better to provide this information before the procedure. However, I would prefer redoing it for both EBS and Cinder together in a separate PR.